kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/KFuzzConfig.kt

```diff
@@ -200,6 +188,9 @@ internal class KFuzzConfigProperty<T : Any> internal constructor(
     }
 }
 
+fun KFuzzConfig.methodReproducerPath(method: Method): Path =
```

Comment:
1. Mb, `reproducerPathOf` is better?
2. It, most likely, should be placed in `engine`. It doesn't look like a function for the end user

Reply:
Fixed